package.json

```diff
@@ -34,12 +36,15 @@
     "nodemon": "^1.10.2",
     "null-loader": "^0.1.1",
     "pre-push": "^0.1.1",
-    "react-addons-test-utils": "^15.4.1",
+    "react": "^15.4.2",
```

Comment:
Now some deps are duplicated in both `dependencies` and `devDependencies`. Is there a reason to do that?